linked_list/LinkedList.c

```diff
@@ -0,0 +1,175 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct Node {
+	int d;			//integer type data
+	struct Node* next;	// pointer to same structure
+};
+
+//  Adding at Front
+void addFront(struct Node **start, int data) {
+	struct Node* nw_node = (struct Node*) malloc(sizeof(struct Node));	//  nw_node a Node to be added
+	nw_node->d = data;		// data
+	if (*start == NULL) {		// If list is empty make it first Node
+		nw_node->next = NULL;
+		*start = nw_node;
+	} else {
+		nw_node->next = *start;	// Else add address of first node to nw_node and make it first node i.e.'start'
+		*start = nw_node;
+	}
+}
+
+//  Adding at last
+void addLast(struct Node **start, int data) {
+	struct Node* nw_node = (struct Node*) malloc(sizeof(struct Node));	// nw_node a node to be added
+	struct Node* temp;		//  'temp' temporary node to traverse to last
+	nw_node->next = NULL;		//  Last element must have null value in its next variable
+	nw_node->d = data;		//  data
+	if (*start == NULL) {
+		*start = nw_node;
+	} else {
+		temp = *start;
+		while (temp->next != NULL) {	// Loop until we get last node
+			temp = temp->next;
+		}
+		temp->next = nw_node;		// Assiging address of nw_node to next variable of last element of list
+	}
+}
+
+//  Adding at given position
+void addAtPos(struct Node **start, int data, int pos) {
```

Comment:
Make insertion operations return the status code for insertion - `1` for success and `0` for failure.